doc/dev/installation.rst

```diff
@@ -114,7 +114,7 @@ including all its submodules:
 
 .. code-block:: console
 
-  git clone --recurse-submodules -j8 https://github.com/PennyLaneAI/catalyst.git
+  git clone --recurse-submodules --shallow-submodules -j2 https://github.com/PennyLaneAI/catalyst.git
```

Comment:
Nice catch! 

My only comment is about the number of git cloning workers. What do you think about the following suggestion? 

```suggestion
  git clone --recurse-submodules --shallow-submodules -j auto https://github.com/PennyLaneAI/catalyst.git
```

Reply:
Sure if auto works that sounds good :) I set it to two since we have two submodules and the docs say (no mention of auto):
```
-j <n>
--jobs <n>
  The number of submodules fetched at the same time. Defaults to the submodule.fetchJobs option.
```

Reply:
My recommendation was based on the git-submodule doc and I was expecting to see more jobs than 2 in general. 

```
-j <n>
--jobs <n>
This option is only valid for the update command. Clone new submodules in parallel with as many jobs. Defaults to the submodule.fetchJobs option.
```
The current version looks fine :+1: